paper-input.html

```diff
@@ -64,6 +64,28 @@
 @element paper-input
 @hero hero.svg
 @demo demo/index.html
+
+### mixins
+
+--paper-input-container-ms-clear  | Mixin applied to the Internet Explorer reveal button -aka the eyeball | {}
```

Comment:
I think just adding this under `### Styling` without the example is enough

Reply:
Awesome thanks,
Still learning the process